starter-code/styles/base.css

```diff
@@ -1,11 +1,16 @@
 /* TODO: Add a rule for images, so they never exceed their container's width. */
 
+.images {
```

Comment:
Though this works for this specific image, you actually want to target all images, otherwise you'd have to give every image the same class.